tests/test_config.cpp

```diff
@@ -288,3 +290,157 @@ TEST_CASE("CircuitConfig") {
         }
     }
 }
+
+TEST_CASE("SimulationConfig") {
+    SECTION("Simple") {
+        const auto config = SimulationConfig::fromFile("./data/config/simulation_config.json");
+        CHECK_NOTHROW(config.getRun());
+        CHECK(config.getRun().tstop == 1000.f);
+        CHECK(config.getRun().dt == 0.025f);
```

Comment:
Conscious choice not to make these fuzzy comparisons?

Reply:
Sorry, I don't understand the comment.
The checks are added to reach 100% of test coverage and to make sure the correct data member is filled with the correct field from the JSON file

Reply:
This is `catch2`, right?

Something like
```suggestion
        REQUIRE_THAT(config.getRun().tstop, WithinULP(1000.f, 1));
        REQUIRE_THAT(config.getRun().dt, WithinULP(0.025f, 1));
```
would be more generic and not rely on the numbers being tested being a power of two.

Needs
```
using Catch::Matchers::WithinULP;
```
somewhere…

Reply:
Ah I understand now. Ill make the change, thanks!